### ISSUE TITLE
Separate stroke and fill

### ISSUE THREAD

**Issue by alercah** [james-j-obrien/bevy_vector_shapes#36]
Currently for stroke and fill I'm making two shapes, one hollow, one not.

It would be nice to avoid duplication and have these be a single entity.

**Comment by musjj** [comment]
Yeah, this would be useful for making shapes with different stroke and fill colors!

**Comment by mirsella** [comment]
+1, it would allow to have semi circle, as of now an arc with hollow = false will overdraw:
![Image](https://github.com/user-attachments/assets/9d19584c-582a-44cf-9c11-e95512b52173)
this is a arc with start 0., and end PI, so a 180 degree, but the internal filling is too big and draw too much
